shopinvader_sale_customer_info/README.rst

```diff
@@ -0,0 +1,70 @@
+==============================
+Shopinvader Sale Customer Info
+==============================
```

Comment:
This file is based under OCA template but shopinvader repo are not under OCA umbrella.
Could you please adapt its content to fit other shopinvader addons ?